editor/src/clj/editor/prefs_dialog.clj

```diff
@@ -95,14 +95,15 @@
             :prefs [{:label "Enable Texture Compression" :type :boolean :key "general-enable-texture-compression" :default false}
                     {:label "Escape Quits Game" :type :boolean :key "general-quit-on-esc" :default false}
                     {:label "Track Active Tab in Asset Browser" :type :boolean :key "asset-browser-track-active-tab?" :default false}
-                    {:label "Path to custom keymap" :type :string :key "custom-keymap-path" :default ""}]}
+                    {:label "Path to custom keymap" :type :string :key "custom-keymap-path" :default ""}
+                    {:label "Code editor font (requires restart)" :type :string :key "code-editor-font-name" :default "Dejavu Sans Mono"}]}
```

Comment:
There's probably a clever way to make the code editor view recreate itself when a user preference is changed, but I do not know how.